src/api/test/functional/webui/search_controller_test.rb

```diff
@@ -112,13 +112,22 @@ def test_header_search_functionality
     page.must_have_text(/Base.* distro without update project/)
   end
 
+  def test_search_by_request_number
+    visit root_path
+    fill_in 'search', with: '#1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.